### ISSUE TITLE
Multiple Payment Method For One Offer (Cloning offers)

### ISSUE THREAD

**Issue by HardenedSteel** [haveno-dex/haveno#1054]
15% security deposit limits traders to have maximum 6 offers at same time (100/15 = 6.66) otherwise makers have to lower limits which is resulting in very low liquidity.

Similar proposals for Bisq:
https://github.com/bisq-network/proposals/issues/402
https://github.com/bisq-network/proposals/issues/288

https://bisq.wiki/Cloning_an_offer

**Comment by monerobull** [comment]
I don't like this for 3 reasons:

1. Essentially makes offers fractionally reserved and the orderbook stats massively inflated.
2. Leads to more failed offers and frustration on the taker side (long roundtrip times mean it will happen that someone takes a fractional reserved offer that is already taken by someone else. The bigger haveno gets, the more this will happen, especially if only a handful of makers fill most of the orderbook.) 
3. Looks like this is a massive change to the underlying protocol, which already isn't that stable right now.

**Comment by boldsuck** [comment]
> 2\. Leads to more failed offers and frustration on the taker side (long roundtrip times mean it will happen that someone takes a fractional reserved offer that is already taken by someone else. The bigger haveno gets, the more this will happen, especially if only a handful of makers fill most of the orderbook.)

Yes, I've had this happen a few times in the current trade protocol on bisq: I want to accept an offer and it's already been taken by someone else.

**Comment by HardenedSteel** [comment]
> fractionally reserved

what? no, wallets are non-custodial.
> orderbook stats

Can be adjusted.
> Leads to more failed offers

Probably can be fixed.

> frustration on the taker side

both taker and maker already frustrated from liquidity issues on both Haveno and Bisq

**Comment by monerobull** [comment]
> fractionally reserved

The offers themselves are factually fractionally reserved. If I put 10 XMR of orders on the books but in reality there is only 1 XMR up for trade, that is fractionally reserved.

> Leads to more failed offers - Probably can be fixed.

Slow roundtrip is a tor limitation, no fixing that. This system worked on localmonero because they have a centralized server and can instantly close offers once one is taken, doesn't work for Haveno.

>both taker and maker already frustrated from liquidity issues on both Haveno and Bisq

bisq already implements fractional reserved offers and it didn't bring them a whole lot more liquidity. It was claimed paypal would massively increase haveno volume, yet i have seen barely any paypal trades so far.

**Comment by boldsuck** [comment]
> both taker and maker already frustrated from liquidity issues on both Haveno and Bisq

It depends on the currency, most: XMR, EUR, USD, BRL
January 1, 2024 - today = 1.223,743 BTC or `72.966.613 USD`
I think that's not a small amount considering that this is a community project.

**Comment by HardenedSteel** [comment]
> If I put 10 XMR of orders on the books but in reality there is only 1 XMR up for trade

as i said order book stats can be adjusted and make it transparent.

> This system worked on localmonero because they have a centralized server and can instantly close offers once one is taken, doesn't work for Haveno.

It shouldn't mean we should give up and use CEX.

> bisq already implements fractional reserved offers

where? and if your understanding of fractional reserved is this, then its good to have it

**Comment by boldsuck** [comment]
> where? and if your understanding of fractional reserved is this, then its good to have it

https://github.com/haveno-dex/haveno/issues/1072#issuecomment-2187143048

**Comment by HardenedSteel** [comment]
Thanks that feature is also really needed for Haveno imo

**Comment by monerobull** [comment]
> It shouldn't mean we should give up and use CEX.

There is no "giving up and using CEX" here, we just don't want fractional reserve offers that will lead to tons of offers constantly failing to take. 

>that feature is also really needed for Haveno imo

And imo it would greatly hurt Haveno and make it even worse of an experience to use for regular users. The only people who would benefit from this are traders with little Monero liquidity. 

I do not support subsidizing these people, you want a solution? **Buy some more Monero cheapcels**.

**Comment by HardenedSteel** [comment]
>  you want a solution? Buy some more Monero cheapcels.

thanks for letting me know you're trolling

**Comment by XMRfamily** [comment]
What about 1 offer but can have multiple payment methods and price tiers (1-5k 3% 5k-10k 2% 10k+ 1%) etc

**Comment by XMRZombie** [comment]
"Looks like this is a massive change to the underlying protocol, which already isn't that stable right now."
A massive change sure, but i dont think conditional offers will make the protocol unstable.
Most important is the taker should use one currency/payment-method only (not using all conditions to trade)
